wordpress-mobile/gutenberg-mobile
# Updated react-native-recyclerview-list hash to include selection fix

Fixes #267  for `PlainText` blocks on Android.

[EDITED]
~was: "fixes #135  for both `RichText` and `PlainText` blocks on Android", updated to reflect as per https://github.com/wordpress-mobile/gutenberg-mobile/pull/265#issuecomment-440796735.~

See https://github.com/wordpress-mobile/react-native-recyclerview-list/pull/11  for more information.

To test:
Before:
1. open the demo app
2. try to select text on ~RichText and~ PlainText-backed blocks (~Header, Paragraph are rich-text~, while Code and Read More are based on PlainText)
3. observe you can't select text (contextual menu doesn't appear, can't copy, etc.)

![gutenberg_selection_works_not_ok](https://user-images.githubusercontent.com/6597771/48850638-e26d9380-ed87-11e8-9927-e3c591b7ed65.gif)


After:
1. open the demo app
2. try to select text on ~RichText and~ PlainText-backed blocks (~Header, Paragraph are rich-text~, while Code and Read More are based on PlainText)
3. observe you can select, copy, paste and such.

![gutenberg_offscreen_works_ok](https://user-images.githubusercontent.com/6597771/48850667-ec8f9200-ed87-11e8-8a61-115534f54574.gif)
